elastic/kibana
# [Synthetics] project monitors - disable deletion and allow enabling/disabling

## Summary

Resolves https://github.com/elastic/kibana/issues/134575
Resolves https://github.com/elastic/kibana/issues/134598

Allows enabling/disabling of project monitors, and disables deletion of project monitors.

<img width="410" alt="Screen Shot 2022-06-16 at 3 19 47 PM" src="https://user-images.githubusercontent.com/11356435/174300541-278d216e-50da-4b1c-a31d-09d63696377f.png">
<img width="377" alt="Screen Shot 2022-06-16 at 3 19 42 PM" src="https://user-images.githubusercontent.com/11356435/174300542-de6ebcfd-43d1-45ce-8b98-f42a6afcff67.png">

Testing
--

1. Create an api key on the Uptime monitor management page. 
2. Create project monitors by running
```
node dist/cli.js push examples/todos/basic.journey.ts --auth [apiKey]  --url [kibanaProject] --project test
```
3. Navigate to the Uptime Monitor List. 
4. Ensure that the delete button is disabled for project monitors. 
5. Ensure that the monitor can be disabled via the enabled toggle
6. Run the project monitors command again. Ensure that the monitors have been re-enabled.
